README.md

```diff
@@ -6,32 +6,34 @@ This is a list of participants to the hacktoberfest event focused in readers of
+| Niharika Krishnan    | [niharikakrishnan](https://github.com/niharikakrishnan) | Python, C++, Java                       |
+| Ashil Shah           | [ashil30](https://github.com/ashil30)                   | c, c++, kotlin                          |
+| Heather Christman    | [efl7a](https://github.com/efl7a)                       | javascript, html, css, ruby             |
+| Jen Strickland       | [jenstrickland](https://github.com/jenstrickland)       | html, css, js, php, ruby, python        |
+| Charitha Rajapakse   | [Charitha96](https://github.com/Charitha96)             | Java, C++, Javascript, html, css        |
+| Tom Wilkinson        | [TomEWilkinson](https://github.com/TomEWilkinson/)      | php, c#, javascript, html, css, Java    |
+| Anmesh Choudhury     | [anmesh-vicky](https://github.com/anmesh-vicky)         | Python, C++, C,html,css,Javascript      |
+| João Marcelo         | [jmarcelinho](https://github.com/jmarcelinho)           | Python, C++, Java, C                    |
+| Niklas Lang          | [nl5001](https://github.com/nl5001)                     | Python, HTML, CSS, Javascript, C#       |
+| Himanshu Gautam      | [himanshu](https://github.com/himanshu81494)            | Python, javascript, C, PHP              |
+| Satyendra singh      | [Satyendra](https://github.com/satysingh87)             | php, c++, python, javascript, html, css |
+| Felix Guzman         | [felixalguzman](https://github.com/felixalguzman)       | C, C++, java                            |
+| Vaibhav Naik         | [Vaibhav](https://github.com/vaibhavnaikprojects)       | java, python, javascript, html, css     |
+| Manav Mehra          | [Manav](https://github.com/manavmehra96)                | Python , C++ , Arduino , Raspberry      |
+| Priyadarshan Singh   | [PDROJACK](https://github.com/PDROJACK)                 | python, c++ , java ,javascript ,html    |
+| Arfad Kallil.        | [arfu786](https://github.com/arfu786).                  | C,C++,HTML,python                       |
+| Prabhat tyagi        | [stebinben](https://github.com/stebinben)               | Python,Java,c                           |
+| Prakhar Sapre        | [prakhar10](https://github.com/prakhar10)               | Java, HTML, CSS, Javascript, Ruby       |
+| Tom Schmidt          | [sittingpilgrim](https://github.com/sittingpilgrim)     | html, css, javascript, SQL, PHP         |
+| Santiago Caamaño     | [SantiCF](https://github.com/SantiCF)                   | java, python, javascript, html, css     |
+| Nevil Vekariya       | [nevil9]((https://github.com/nevil9)                    | c, cpp, java,html,python,ruby,          |
+| Aryan Patel          | [aryan411]((https://github.com/aryan411)                | c, cpp                                  |
+| Josh Cook            | [Joshua]((https://github.com/cookiej91)                 | javascript, SQL, Swift                  |
```

Comment:
Looks like this link is broken, should be `[Joshua](https://github.com/cookiej91)` (one opening bracket).